docs/compression_algorithms/Pruning.md

```diff
@@ -69,6 +69,18 @@ and reduce the corresponding accuracy drop even before model training. This opti
 sparsity and filter pruning algorithms. It can be enabled by setting a non-zero value of `num_bn_adaptation_samples` in the
 `batchnorm_adaptation` section of the `initializer` configuration (see example below).
 
+#### Interlayer ranking types
+
+
+Interlayer ranking type can be one of `unweighted_ranking` or `learned_ranking`.  
+ In case of `unweighted_ranking` and with  
```

Comment:
Remove this extra line break

Reply:
@mkaglins, please take a look at the rendered markdown. The current formatting of this paragraph looks ugly. Please fix.

Reply:
Done